samples/templates/express-direct-auth/web-server/routes/recover-password.js

```diff
@@ -30,6 +30,13 @@ router.post('/recover-password', async (req, res, next) => {
     username,
     authenticators: authenticator ? [authenticator] : [],
   });
+  const { messages } = transaction;
+  if (messages && messages.length) {
```

Comment:
https://github.com/okta/okta-auth-js/blob/master/samples/generated/express-direct-auth/web-server/utils/handleTransaction.js#L23

Can we make change here to add case for `identify-recovery`? Initially, I was thinking to add some composed switch functions logic to handle the next pending steps, but thought it might be overkill.